Sources/Plasma/Apps/plPythonPack/main.cpp

```diff
@@ -213,18 +213,19 @@ void FindFiles(std::vector<plFileName> &filenames, std::vector<plFileName> &path
     // Get the names of all the python files
     std::vector<plFileName> pys = plFileSystem::ListDir(path, "*.py");
 
-    for (auto iter = pys.begin(); iter != pys.end(); ++iter)
+    for (const auto& py : pys)
     {
-        filenames.push_back(iter->GetFileName());
+        filenames.push_back(py.GetFileName());
         pathnames.push_back(path);
     }
 }
 
 void FindSubDirs(std::vector<plFileName> &dirnames, const plFileName &path)
 {
     std::vector<plFileName> subdirs = plFileSystem::ListSubdirs(path);
-    for (auto iter = subdirs.begin(); iter != subdirs.end(); ++iter) {
-        ST::string name = iter->GetFileName();
+    for (const auto& subdir : subdirs)
+    {
+        ST::string name = subdir.GetFileName();
```

Comment:
```suggestion
        const ST::string& name = subdir.GetFileName();
```

Let's defer the copy until we're really sure that we need it.